concent_api/core/tests/test_core_views.py

```diff
@@ -558,34 +556,33 @@ class CoreViewReceiveTest(ConcentIntegrationTestCase):
     def setUp(self):
         with freeze_time("2017-11-17 10:00:00"):
             super().setUp()
-            self.compute_task_def = ComputeTaskDefFactory()
+            self.compute_task_def = tasks.ComputeTaskDefFactory()
             self.compute_task_def['deadline'] = get_current_utc_timestamp() + (60 * 37)
-            self.want_to_compute_task = WantToComputeTaskFactory(
+            self.want_to_compute_task = tasks.WantToComputeTaskFactory(
```

Comment:
are all those params below necessary? If not, we could reduce the boilerpalte ;)

Reply:
That's true, deleted :)